libraries/tools/kotlin-stdlib-gen/src/templates/Arrays.kt

```diff
@@ -443,7 +444,17 @@ object ArrayOps : TemplateGroupBase() {
         include(ArraysOfPrimitives, PrimitiveType.defaultPrimitives)
         include(InvariantArraysOfObjects)
     } builder {
-        doc { "Returns new array which is a copy of the original array, resized to the given [newSize]." }
+        doc {
+            """
+            Returns new array which is a copy of the original array, resized to the given [newSize], truncating or padding
+            with nulls if necessary. If the object is a primitive type, it will be padded with its default value.
```

Comment:
Please add a blank line to separate summary and description.
The text can be specialized for primitives based on the current value of overloads `family`.

Reply:
About text specialization, I am not sure how to approach this without coping description, the only thing I found is `textWhen` method, but with it description looks messy.

```
            """
            Returns new array which is a copy of the original array, resized to the given [newSize],
            truncating or padding with ${textWhen(family == ArraysOfPrimitives) { "primitive default value" }}${textWhen(family == InvariantArraysOfObjects) { "null" }} if necessary.

            For all indices that are valid in both the original array and the copy, the two arrays contents will be identical.
            For all indices that are valid in the copy, but not in the original, the copy will contain ${textWhen(family == ArraysOfPrimitives) { "primitive default value" }}${textWhen(family == InvariantArraysOfObjects) { "null" }}.
            """
```

Reply:
If `textWhen` is messy, you can use plain `if` or `when` expression inside `${...}`